frontend/sozisel-app/src/components/SessionDetails/SessionCustomization/SessionCustomization.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../../styles/styles";
+
+.sessionDetailsContainer {
```

Comment:
start top level scss class with a capital letter and the inner classes with a lower letter